duckduckgo/Android
# Internal setting to override UA

Task/Issue URL: https://app.asana.com/0/42792087274227/1201464130371298/f

### Description
This PR adds the capability to override app UA on internal builds for testing purposes.

### Steps to test this PR
_Play debug_
1. Install play build
2. ensure no dev settings are visible
5. go to serp
6. search for "what's my user agent"
7. Ensure UA detected is the expected one ([more info inside Asana task](https://app.asana.com/0/42792087274227/1201464130371298/f))

_internal debug_
1. Install internal build
2. Go to dev settings
3. toggle on override UA
4. select any UA from the list
5. go to serp
6. search for "what's my user agent"
7. Ensure UA detected is the expected one 

### UI changes
| Before  | After |
| ------ | ----- |
!(Upload before screenshot)|(Upload after screenshot)|
